nevergrad/optimization/recorded_recommendations.csv

```diff
@@ -84,6 +86,8 @@ MultiCMA,1.012515477,-0.9138805701,-1.029555946,1.2098418178,,,,,,,,,,,,
 MultiScaleCMA,0.0002149759,-0.0003843636,-0.0002539104,7.32548e-05,,,,,,,,,,,,
 MutDE,1.1400386808,0.3380024444,0.4755144618,2.6390460807,0.6911075733,1.111235567,-0.2576843178,-1.1959512855,,,,,,,,
 NGO,0.0,-0.3451057176,-0.1327329683,1.9291307781,,,,,,,,,,,,
+NaiveAnisoEMNA,1.012515477,-0.9138691467,-1.0295302074,1.2097964496,,,,,,,,,,,,
+NaiveEMNA,1.012515477,-0.9138691467,-1.0295302074,1.2097964496,,,,,,,,,,,,
```

Comment:
Please merge master and then rerun pytest to record the new values. Since names have changed this should be updated. Hopefully that should solve the continuous integration issues